tests/components/airzone/test_select.py

```diff
@@ -0,0 +1,177 @@
+    """Test creation of selects."""
+
+    await async_init_integration(hass)
+
+    state = hass.states.get("select.despacho_cold_angle")
+    assert state.state == "90º"
+
+    state = hass.states.get("select.despacho_heat_angle")
+    assert state.state == "90º"
+
+    state = hass.states.get("select.despacho_sleep")
+    assert state.state == "Off"
+
+    state = hass.states.get("select.dorm_1_cold_angle")
+    assert state.state == "90º"
+
+    state = hass.states.get("select.dorm_1_heat_angle")
+    assert state.state == "90º"
+
+    state = hass.states.get("select.dorm_1_sleep")
+    assert state.state == "Off"
+
+    state = hass.states.get("select.dorm_2_cold_angle")
+    assert state.state == "90º"
+
+    state = hass.states.get("select.dorm_2_heat_angle")
+    assert state.state == "90º"
+
+    state = hass.states.get("select.dorm_2_sleep")
+    assert state.state == "Off"
+
+    state = hass.states.get("select.dorm_ppal_cold_angle")
+    assert state.state == "45º"
+
+    state = hass.states.get("select.dorm_ppal_heat_angle")
+    assert state.state == "50º"
+
+    state = hass.states.get("select.dorm_ppal_sleep")
+    assert state.state == "30m"
+
+    state = hass.states.get("select.salon_cold_angle")
+    assert state.state == "90º"
+
+    state = hass.states.get("select.salon_heat_angle")
+    assert state.state == "90º"
+
+    state = hass.states.get("select.salon_sleep")
+    assert state.state == "Off"
+
+
+async def test_airzone_select_sleep(hass: HomeAssistant) -> None:
+    """Test select sleep."""
+
+    await async_init_integration(hass)
+
+    HVAC_MOCK = {
+        API_DATA: [
+            {
+                API_SYSTEM_ID: 1,
+                API_ZONE_ID: 3,
+                API_SLEEP: 30,
+            }
+        ]
+    }
+
+    with pytest.raises(ValueError):
+        await hass.services.async_call(
+            SELECT_DOMAIN,
+            SERVICE_SELECT_OPTION,
+            {
+                ATTR_ENTITY_ID: "select.dorm_1_sleep",
+                ATTR_OPTION: "Invalid",
+            },
+            blocking=True,
+        )
+
+    with patch(
+        "homeassistant.components.airzone.AirzoneLocalApi.put_hvac",
+        return_value=HVAC_MOCK,
+    ):
+        await hass.services.async_call(
+            SELECT_DOMAIN,
+            SERVICE_SELECT_OPTION,
+            {
+                ATTR_ENTITY_ID: "select.dorm_1_sleep",
+                ATTR_OPTION: "30m",
+            },
+            blocking=True,
+        )
+
+    state = hass.states.get("select.dorm_1_sleep")
+    assert state.state == "30m"
+
+
+async def test_airzone_select_grille_angle(hass: HomeAssistant) -> None:
+    """Test select sleep."""
+
+    await async_init_integration(hass)
+
+    # Cold Angle
+
+    HVAC_MOCK = {
+        API_DATA: [
+            {
+                API_SYSTEM_ID: 1,
+                API_ZONE_ID: 3,
+                API_COLD_ANGLE: 1,
+            }
+        ]
+    }
```

Comment:
Please lowercase the var name since you aren't using it as a constant at the top of the file (or inline it in your `patch` since you don't reuse it.)
```suggestion
    put_hvac_return = {
        API_DATA: [
            {
                API_SYSTEM_ID: 1,
                API_ZONE_ID: 3,
                API_COLD_ANGLE: 1,
            }
        ]
    }
```

Reply:
Fixed in 7e35dbc38cb4063c862ff25c5fd5e58b8e96fc80